### PR TITLE
Added youtube_player

### PR DESCRIPTION
**Pull request by fullstackctp**
Ticket #655

### CODE REVIEW

File: lib/widget/ensemble_player.dart [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import 'package:ensemble/framework/widget/widget.dart';
+import 'package:ensemble/util/utils.dart';
+import 'package:ensemble/widget/helpers/controllers.dart';
+import 'package:ensemble_ts_interpreter/invokables/invokable.dart';
+import 'package:flutter/material.dart';
+import 'package:youtube_player_iframe/youtube_player_iframe.dart';
+
+class EnsemblePlayer extends StatefulWidget
```
Comment (vusters) [inline lib/widget/ensemble_player.dart:8]: name it the same name as what you have in the YAML.

File: lib/widget/ensemble_player.dart [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        videoId: widget._controller.link.split("=").last,
+      );
+    }
+    super.initState();
+  }
+
+  @override
+  Widget buildWidget(BuildContext context) {
+    return YoutubePlayerScaffold(
+        aspectRatio: widget._controller.aspectRatio ?? 16 / 9,
```
Comment (vusters) [inline lib/widget/ensemble_player.dart:62]: Does it auto-detect aspect ratio? If so don't set a default.

**Comment by kmahmood74** [comment]
@fullstackctp what's the status of this one? Vu commented last week. Have you made the changes

**Comment by fullstackctp** [comment]
I have made the required changes and also added all the features that i demonstrated during the stand up.

**Review by kmahmood74 — APPROVED** [review]
I trust you tested it across all devices and have made the changes I requested. Feel free to commit

**Comment by kmahmood74** [comment]
approved, please merge and commit asap

**Comment by kmahmood74** [comment]
@fullstackctp this has been approved, please merge and commit

**Pull request merged**